sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -614,7 +628,10 @@ public virtual async Task StartProcessingAsync(CancellationToken cancellationTok
         ///
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> instance to signal the request to cancel the stop operation.  If the operation is successfully canceled, the <see cref="EventProcessorClient" /> will keep running.</param>
         ///
-        public virtual async Task StopProcessingAsync(CancellationToken cancellationToken = default)
+        public virtual void StopProcessing(CancellationToken cancellationToken = default)
+            => StopProcessingImplAsync(false, cancellationToken).EnsureCompleted();
```

Comment:
If we remove this new method, couldn't we just do `StopProcessingAsync(cancellationToken).EnsureCompleted();`?     I'm not sure of the benefit of refactoring into a separate method.